Twitch EventSub library/CoreFunctions/LoggerExtension.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.Extensions.Logging;
+
+
+namespace Twitch.EventSub.CoreFunctions
+{
+    public static class LoggerExtension
+    {
+        public static ILogger LogDetails(this ILogger logger, LogLevel logLevel, params object[] args)
+        {
+            if (logger == null)
+                throw new ArgumentNullException(nameof(logger));
+
+            if (!logger.IsEnabled(logLevel))
+                return logger;
+
+            var message = string.Join(Environment.NewLine, args.OfType<string>());
```

Comment:
Careful here, `args` can be null. If you pass no objects (plain text log), it will crash here (and on following lines)